src/components/AdminSettings.vue

```diff
@@ -1,3 +1,25 @@
+<!--
+  - @copyright Copyright (c) Matthias Heinisch <nextcloud@matthiasheinisch.de>
+  -
+  - @author Matthias Heinisch <nextcloud@matthiasheinisch.de>
```

Comment:
:+1:

Reply:
so you know who to blame :man_juggling: :policeman: